engine/main.go

```diff
@@ -280,49 +278,67 @@ See $ engine config command for more details.
 			signal.Stop(c)
 		}()
 
-		for _, a := range args {
-			var s Service
-			var cfg interface{}
+		type ServiceConf struct {
```

Comment:
what does it stand for ? why is it public ?

Reply:
public -> private
this struct is only here to collect args, to go start each service below. We start services, after init log (with name field with all names concat with '_')